test-configs.yaml

```diff
@@ -1030,11 +1020,76 @@ device_types:
     filters:
       - whitelist: {defconfig: ['defconfig']}
 
+  r8a7743-iwg20d-q7:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    dtb: 'r8a7743-iwg20d-q7-dbcm-ca.dtb'
+    filters:
+      - whitelist: {defconfig: ['multi_v7_defconfig', 'shmobile_defconfig']}
+
+  r8a7745-iwg22d-sodimm:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    dtb: 'r8a7745-iwg22d-sodimm-dbhd-ca.dtb'
+    filters:
+      - whitelist: {defconfig: ['multi_v7_defconfig', 'shmobile_defconfig']}
+
+  r8a77470-iwg23s-sbc:
```

Comment:
There's a problem with this device type, it's failing to boot with a ramdisk, both in health checks and in kernelci jobs:
https://lava.ciplatform.org/scheduler/job/19319
https://lava.ciplatform.org/scheduler/device_type/r8a77470-iwg23s-sbc?dt_length=25&dt_search=kernelci#dt_

So it's probably best to drop it from this PR initially, to only add devices that are known to be working well, and add it when the issue has been fixed (upstream kernel? bootloader?).

Reply:
I'm happy to drop this device for now. Please double check that the other devices are working okay for you though.